magento/pwa-studio
# [PWA-1543] Verify Divider

## Description

To verify that the Page Builder Divider content type renders correctly on Venia for each content type property.

## Related Issue





Closes [PWA-1543](https://jira.corp.magento.com/browse/PWA-1543): Verify Divider

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Cypress `divider.spec.js` should pass
Steps - 
Go to venia-integration-test dir
Run `yarn install`
Run `yarn test`
Run `divider.spec.js` from cypress ui

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
